<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-11-15T18:54:20 IST" name="Default suite" finished-at="2024-11-15T19:01:57 IST" duration-ms="456970">
    <groups>
    </groups>
    <test started-at="2024-11-15T18:54:20 IST" name="Default test" finished-at="2024-11-15T19:01:57 IST" duration-ms="456970">
      <class name="Selenium.Practice.demo2">
        <test-method is-config="true" signature="beforesuit()[pri:0, instance:Selenium.Practice.demo2@202b0582]" started-at="2024-11-15T18:54:20 IST" name="beforesuit" finished-at="2024-11-15T18:54:20 IST" duration-ms="297" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforesuit -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:Selenium.Practice.demo2@202b0582]" started-at="2024-11-15T18:54:20 IST" name="beforeMethod" finished-at="2024-11-15T18:54:25 IST" duration-ms="5096" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="test2()[pri:0, instance:Selenium.Practice.demo2@202b0582]" started-at="2024-11-15T18:54:25 IST" name="test2" finished-at="2024-11-15T19:01:57 IST" duration-ms="451382" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[login failed for users: [awarehime@morstan.com, agoldfarb@morstan.com, jabney@neccsouth.com, arosenberg@morstan.com, amakabali@morstan.com, ARamsey@Morstan.com, jbond@morstan.com, msilver@morstan.com, pablo.zapata@hullco.com, pfitzgerald@morstan.com, rose.bates@hullco.com]]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: login failed for users: [awarehime@morstan.com, agoldfarb@morstan.com, jabney@neccsouth.com, arosenberg@morstan.com, amakabali@morstan.com, ARamsey@Morstan.com, jbond@morstan.com, msilver@morstan.com, pablo.zapata@hullco.com, pfitzgerald@morstan.com, rose.bates@hullco.com]
at junit.framework.Assert.fail(Assert.java:47)
at Selenium.Practice.demo2.test2(demo2.java:115)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:Selenium.Practice.demo2@202b0582]" started-at="2024-11-15T19:01:57 IST" name="afterMethod" finished-at="2024-11-15T19:01:57 IST" duration-ms="449" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test2 status=FAILURE method=demo2.test2()[pri:0, instance:Selenium.Practice.demo2@202b0582] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="aftersuit()[pri:0, instance:Selenium.Practice.demo2@202b0582]" started-at="2024-11-15T19:01:57 IST" name="aftersuit" finished-at="2024-11-15T19:01:58 IST" duration-ms="377" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftersuit -->
      </class> <!-- Selenium.Practice.demo2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
